File: bl-central-server/bl-sensor-data-collector/src/main/java/org/jesperancinha/logistics/sensor/collector/data/TrainLogDto.java
package org.jesperancinha.logistics.sensor.collector.data;

import java.math.BigDecimal;

public record TrainLogDto(Long id,
    String source,
    String type,
    Long timestamp,
    BigDecimal lat,
    BigDecimal lon,
    Long weight,
    Long carriageId) {
}

File: bl-central-server/bl-sensor-data-collector/src/main/kotlin/org/jesperancinha/logistics/sensor/collector/data/TrainLogDto.kt
package org.jesperancinha.logistics.sensor.collector.data

import java.math.BigDecimal
import java.util.UUID

data class TrainLogDto(
    val id: UUID,
    val source: String,
    val type: String,
    val timestamp: Long,
    val lat: BigDecimal,
    val lon: BigDecimal,
    val weight: Long,
    val carriageId: UUID
)